rts/idris_utf8.c

```diff
@@ -78,6 +78,36 @@ unsigned idris_utf8_index(char* s, int idx) {
    return top;
 }
 
+char* idris_utf8_advance(char* str, int i) {
```

Comment:
Isnt there the option to get the byte length of the buffer from the idris side?

> Yeah, I noted that in the last review.

Sorry for the duplication.